package.json

```diff
@@ -5,13 +5,13 @@
     "packages/*"
   ],
   "devDependencies": {
-    "@deck.gl/carto": "8.5.0-beta.1",
-    "@deck.gl/core": "8.5.0-beta.1",
-    "@deck.gl/extensions": "8.5.0-beta.1",
-    "@deck.gl/geo-layers": "8.5.0-beta.1",
-    "@deck.gl/google-maps": "8.5.0-beta.1",
-    "@deck.gl/layers": "8.5.0-beta.1",
-    "@deck.gl/mesh-layers": "8.5.0-beta.1",
+    "@deck.gl/carto": "8.5.0",
+    "@deck.gl/core": "8.5.0",
+    "@deck.gl/extensions": "8.5.0",
+    "@deck.gl/geo-layers": "8.5.0",
+    "@deck.gl/google-maps": "8.5.0",
+    "@deck.gl/layers": "8.5.0",
+    "@deck.gl/mesh-layers": "8.5.0",
```

Comment:
This just applies to dev environment. You must also update ALL other "@deck.gl/***" references, inside the packages: eg. the deck.gl references in packages/react-api/package.json